gap/oper.gi

```diff
@@ -50,6 +50,49 @@ end);
 
 #
 
+InstallMethod(IsUndirectedSpanningForest, "for a digraph and a digraph",
+[IsDigraph, IsDigraph],
+function(super, sub)
+  local sym, comps1, comps2;
+
+  if not IsUndirectedForest(sub) then
+    return false;
+  fi;
+
+  sym := MaximalSymmetricSubdigraph(super);
+
+  if not IsSubdigraph(sym, sub) then
+    return false;
+  fi;
+
+  comps1 := DigraphConnectedComponents(sym).comps;
+  comps2 := DigraphConnectedComponents(sub).comps;
+
+  if Length(comps1) <> Length(comps2) then
+    return false;
+  fi;
+
+  comps1 := List(comps1, ShallowCopy);
+  comps2 := List(comps2, ShallowCopy);
+  Perform(comps1, Sort);
```

Comment:
I think these 6 lines could be two: 

`comps1 := Set(comps1, SortedList);`

but I didn't test this.